{{ cookiecutter.format }}/{{ cookiecutter.app_name }}.wxs

```diff
@@ -98,13 +98,31 @@
             </Directory>
         </Directory>
 
+        {% if cookiecutter.document_types -%}
+        <SetProperty Id='FileAssociationProperty' Value="[{{ cookiecutter.module_name }}_ROOTDIR]{{ cookiecutter.formal_name }}.exe" After="CostFinalize"/>
+
+        {% for document_type_id, document_type in cookiecutter.document_types|dictsort -%}
+        <Component Id="FileAssociation.{{ document_type_id }}" Directory="{{ cookiecutter.module_name }}_ROOTDIR" Guid="*">
+            <File Id="ProductIcon.{{ document_type_id }}" Source="src\app\{{ cookiecutter.module_name }}\{{ document_type.icon }}.ico" />
```

Comment:
This won't work as defined - there's a couple of problems:

1. The `/` issue identified by @fedelibre 
2. The explicit inclusion of `app` in the path - that presupposes the location of the icon in the app layout. 
3. Briefcase won't copy the icon unless there's a separate `document_type_icon` definition in briefcase.toml as well to tell where the icon should be copied to.

The current setup likely works if you're putting the icon in the app folder, and declaring it using a path that matches the `app` folder. Fixing (3) will solve (1) and (2) as well.

See https://github.com/beeware/briefcase-macOS-app-template/blob/main/%7B%7B%20cookiecutter.format%20%7D%7D/briefcase.toml#L28C1-L30C13 for the declaration that is missing; that will also define the "right" value for this setting.

Reply:
1. Turns out is not an issue, the `\` is used on Windows and the strings aren't Python strings so we don't need to escape the backslashes.
2. Yes, it does. It seemed like the easiest way to find the icon in the build directory since the app is copied over.
3. This would be great! I'll look into this.